<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="root_widget">
   <property name="minimumSize">
    <size>
     <width>415</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="lbl_no_offset">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>No Offset</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_parent_no_offset">
            <property name="text">
             <string>Parent
Constraint</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_point_no_offset">
            <property name="text">
             <string>Point
Constraint</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_orient_no_offset">
            <property name="text">
             <string>Orient
Constraint</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_scale_no_offset">
            <property name="text">
             <string>Scale
Constraint</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="lbl_offset">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>With Offset</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_parent_offset">
            <property name="text">
             <string>Parent
Constraint</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_point_offset">
            <property name="text">
             <string>Point
Constraint</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_orient_offset">
            <property name="text">
             <string>Orient
Constraint</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_scale_offset">
            <property name="text">
             <string>Scale
Constraint</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_remove_constraint">
        <property name="text">
         <string>Remove Constraint</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
